src/Raven.Server/Json/JsonDeserialization.cs

```diff
@@ -94,9 +94,6 @@ private static Expression GetValue(PropertyInfo propertyInfo, ParameterExpressio
                 type == typeof(long) ||
                 type == typeof(int) ||
                 type == typeof(double) ||
-                type == typeof(DateTime) ||
```

Comment:
This is going into a document, which means that it needs to be _readable_

*Hibernating Rhinos Ltd  *

Oren Eini* l CEO l *Mobile: + 972-52-548-6969

Office: +972-4-622-7811 *l *Fax: +972-153-4-622-7811



On Tue, Jul 12, 2016 at 1:28 PM, Fitzchak Yitzchaki <
notifications@github.com> wrote:

> In src/Raven.Server/Json/JsonDeserialization.cs
> <https://github.com/ayende/ravendb/pull/3986#discussion_r70413875>:
>
> > @@ -94,9 +94,6 @@ private static Expression GetValue(PropertyInfo propertyInfo, ParameterExpressio
> >                  type == typeof(long) ||
> >                  type == typeof(int) ||
> >                  type == typeof(double) ||
> > -                type == typeof(DateTime) ||
>
> We'll add it once we'll need it. It should probably read a long value and
> convert it to UTC DateTime. But for now it made the code simple to have the
> ticks directly in the status object.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/ayende/ravendb/pull/3986/files/d2f7422f53856abca48b9fcd0fee45ca62f19d16..f22121a177d00c2633846598e4aa1d96c641c3e4#r70413875>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAHIs4Pl015ikcy5Rg70bdiYp6D4Q0sqks5qU2xlgaJpZM4JKNmp>
> .
>